rollup.config.js

```diff
@@ -14,21 +14,8 @@ const compiledLicense = _.template(fs.readFileSync('./build/license-header.txt',
 const bannerData = _.pick(pkg, ['version', 'copyright']);
 const banner = compiledLicense(Object.assign({includesVtt: true}, bannerData));
 
-const watch = {
-  clearScreen: false
-};
-
-const onwarn = (warning) => {
```

Comment:
This was hiding several warnings that we should have handled.

Reply:
What type of thing specifically? Does adding the globals/externals prevent it from printing import errors but still keeps the same behavior?